specification.md

```diff
@@ -194,6 +208,32 @@ Represents person registration data.
 }
 ```
 
+### RegistrationInfo
+
+Represents information related to when and how to register for the competition.
+
+| Attribute | Type | Description |
+| --- | --- | --- |
+| `openTime` | [`DateTime`](#datetime) | The point in time when online registration opens. |
+| `closeTime` | [`DateTime`](#datetime) | The point in time when online registration closes. |
+| `baseEntryFee` | `Integer` | The competition's base fee for online registration. |
+| `currencyCode` | [`CurrencyCode`](#currencycode) | The currency of the `baseEntryFee`. |
+| `competitorLimit` | `Integer` | The maximum number of competitors who can register for this competition. |
+| `walkInsPermitted` | `Boolean` | Whether non-registered competitors are permitted to sign up at the competition. +
+
+#### Example
+
+```json
+{
+  "openTime": "2023-08-29T05:00:00Z",
+  "closeTime": "2023-11-18T05:00:00Z",
+  "baseEntryFee": 20,
+  "currencyCode": "USD",
+  "competitorLimit": 100,
```

Comment:
Doesn't this conflict with `competitorLimit` being a top level prop?

Reply:
Yes. I mentioned this in the first message of the PR. I think it's not ideal that competitorLimit is a top-level prop -- everything else is nested under sub-properties, and once we have a registrationInfo field it's weird that it won't include competitorLimit. But I'm also not sure that there is a reasonable path to deprecating a field from WCIF (we could set it in both places, then eventually delete the top-level one, but there's no way of communicating with all users of the field to tell them to stop using it. Or we could just accept some breakage). And keeping both fields indefinitely is odd.

What do you think?

Reply:
I'm not a fan of breaking apis ;) AGE broke in the past, not fun.

Currently the endpoint for fetching the WCIF is api/v0/competitions/...
If you want to make a breaking change (e.g. moving a field), then create a api/v1
All current software will keep using the v0 endpoint, and devs who know about v1 and its changes can use the new one

Reply:
Yeah good point!

I'll take competitorLimit out of this object, and file an issue to move it as part of an eventual v1 migration.